pom.xml

```diff
@@ -818,6 +827,7 @@
                     </excludes>
                     <runOrder>random</runOrder>
                     <threadCount>1</threadCount>
+                    <forkCount>0</forkCount>
```

Comment:
Why no fork?

Reply:
Idk, didn't touch this part of the code/build..

Reply:
Merge error maybe?

Reply:
Hmm I can't remember what I did there, can you use the config from master and see if that's working?

Reply:
I made the change in bf54d71

Reply:
The forkCount does not exist on master..

Reply:
Good point! Better to leave it out, than disable it using zero.